test/integration/regression/test_regression_bug390.py

```diff
@@ -0,0 +1,37 @@
+from inspect import cleandoc
+
+pytest_plugins = "pytester"
+
+
+def test_connection_with_block_cleans_up_properly(pytester, exasol_config):
+    config = exasol_config
+    # Because the error only occurs on process shutdown we need to run a test within a test
+    # (We require the result (stderr) of a terminated process triggering the failure.
```

Comment:
Is it necessary to run a test within a test? Would it be also possible to just run a standalone python script in another process and check its stderr?

Reply:
As discussed